.github/workflows/build.yml

```diff
@@ -4,7 +4,7 @@ jobs:
   build:
     strategy:
       matrix:
-        go-version: [1.17.x, 1.18.x
```

Comment:
shouldn't we also check 1.18 ? It's a [supported](https://endoflife.date/go) golang version.

Reply:
> shouldn't we also check 1.18 ? It's a [supported](https://endoflife.date/go) golang version.

I don't know how it will work when go.mod is explicitly set to version 1.19